.github/workflows/build-40-aurora.yml

```diff
@@ -7,12 +7,14 @@ on:
       - testing
     paths-ignore:
       - '**.md'
+      - 'system_files/bluefin/**'
   push:
     branches:
       - main
       - testing
     paths-ignore:
       - '**.md'
+      - 'system_files/bluefin/**'
```

Comment:
shouldn't "bluefin" be "silverblue" in these paths?

Reply:
you're absolutely right.